demo/api/src/db/fixtures/generators/blocks/columns.fixture.ts

```diff
@@ -0,0 +1,61 @@
+import { ExtractBlockInputFactoryProps } from "@comet/blocks-api";
+import { Injectable } from "@nestjs/common";
+import { ColumnsBlock, supportedBlocks } from "@src/pages/blocks/columns.block";
+import { datatype, random } from "faker";
+
+import { ColumnsContentBlock } from "../../../../pages/blocks/columns.block";
+import { DamImageBlockFixtureService } from "./dam-image.fixture";
+import { HeadlineBlockFixtureService } from "./headline.fixture";
+import { BlockFixture } from "./page-content.fixture";
+import { RichTextBlockFixtureService } from "./richtext.fixture";
+import { SpaceBlockFixtureService } from "./space.fixture";
+
+@Injectable()
+export class ColumnsBlockFixtureService {
+    constructor(
+        private readonly richtextBlockFixtureService: RichTextBlockFixtureService,
+        private readonly damImageBlockFixture: DamImageBlockFixtureService,
+        private readonly spaceBlockFixture: SpaceBlockFixtureService,
+        private readonly headlineBlockFixture: HeadlineBlockFixtureService,
+    ) {}
+
+    async generateBlock(): Promise<ExtractBlockInputFactoryProps<typeof ColumnsBlock>> {
+        const layoutTypes = ["one-column", "two-columns"];
+        const columnCount = datatype.number({ min: 1, max: 2 });
+        const layout = layoutTypes[columnCount - 1];
+        const columns = [];
+
+        for (let i = 0; i < columnCount; i++) {
+            const blocks: ExtractBlockInputFactoryProps<typeof ColumnsContentBlock>["blocks"] = [];
+
+            const blockCfg: Record<keyof typeof supportedBlocks, BlockFixture> = {
```

Comment:
I'm not particularly fond of this name. Maybe `supportedBlocksFixtureGenerators`? Also, this could be created in the constructor, and not in every loop